src/components/application_manager/src/commands/mobile/system_request.cc

```diff
@@ -53,12 +56,360 @@ uint32_t SystemRequest::index = 0;
 const std::string kSYNC = "SYNC";
 const std::string kIVSU = "IVSU";
 
+static const char* kQeeryAppsValidationFailedPrefix =
+    ":QUERY_APPS_VALIDATION_FAILED: ";
+
+template <typename ObjectType>
+class QueryAppsDataValidatorTemplate {
+ public:
```

Comment:
@okoreniak, since this class is used only in this cc file then do next
- extract all consts and the validator itsellf to the anon namespace
- I don't see the point in `error_prefix_` member. Turn the kQeeryAppsValidationFailedPrefix  into the const string (same way as it's done for other consts here) and use it directly.
- double check how the private consts should be named. can we make those vars static and init them where they are declared?